src/adapters/adapter.js

```diff
@@ -33,6 +33,8 @@ export default class Adapter extends Configurable {
 
     this.bot = bot;
     this.initUsers();
+    this.prompts = {};
+    this.bot.emitter.on(AO.PROMPT_USER, this.promptUser, this);
```

Comment:
because `this.promptUser` is defined below as `promptUser = () => `, it should already be bound - and you shouldn't have to add `this` at all as the third parameter.